packages/app/src/systems/Asset/hooks/useAsset.tsx

```diff
@@ -2,11 +2,18 @@ import { useMemo } from "react";
 
 import { ASSET_LIST } from "../utils";
 
+import { relativeUrl } from "~/systems/Core/utils/relativeUrl";
+
 export function useAsset(assetId: string) {
-  const asset = useMemo(
-    () => ASSET_LIST.find((asset) => asset.assetId === assetId),
-    [assetId]
-  );
+  const asset = useMemo(() => {
+    const assetItem = ASSET_LIST.find((asset) => asset.assetId === assetId);
+    if (!assetItem) return null;
+
+    // Fix image in according to the baseUrl
+    assetItem.imageUrl = relativeUrl(assetItem.imageUrl);
```

Comment:
I guess should be better:

`return { ...assetItem, imageUrl: relativeUrl(assetItem.imageUrl) }`

instead of mutate `assetItem`